demo/plugins/rpc/README.md

```diff
@@ -14,10 +14,12 @@ To initialize the Controller, run the following commands from the root of this d
 
 ```sh
 cd controller/
-source .venv/bin/activate
 poetry install --no-root --all-extras
+cp .env.example .env
 ```
 
+If you are running Traction in a devcontainer you should adjust the `PROXY_URL` in the `.env` to point to `host.docker.internal` instead of `localhost`.
```

Comment:
For me (on Windows) I had to change to host.docker.internal even though I wasn't running Traction itself in a devcontainer (it's funny, was just sorting through the same thing on an unrelated issue https://github.com/hyperledger/aries-cloudagent-python/issues/2938)

I _think_ that's Windows-docker-specific and not just the setup I'm using so maybe worth a language in this part about "If you are running Traction in a devcontainer **(or on Windows outside WSL)**" or something?

Reply:
Updated for clarity